examples/kusama/assetHub/foreignAssetTransfers/local/assetHubForeignAssetsTransfer.ts

```diff
@@ -28,8 +27,6 @@ const main = async () => {
 			['1000000000000'],
 			{
 				format: 'call',
```

Comment:
```suggestion
				format: 'call',
				xcmVersion: 3
```
This example fails otherwise

Reply:
Great catch! After considering allowing this to be optional I think it makes sense to be explicit so updated the example rather than change it